src/test/data/entity/ccjData.ts

```diff
@@ -0,0 +1,29 @@
+import { MomentFactory } from 'shared/momentFactory'
+
+export const ccjDeterminationByInstalment = {
+  ccjType: 'DETERMINATION',
+  paidAmount: 10,
+  paymentOption: 'INSTALMENTS',
+  repaymentPlan: {
+    paymentLength: '100 months',
+    completionDate: '2028-04-01',
+    paymentSchedule: 'EVERY_MONTH',
+    firstPaymentDate: '2020-01-01',
```

Comment:
can we not do something like this 'MomentFactory.currentDate().add(2, 'year')' as done below for payBySetDate

Reply:
Sadly these dates can't be dynamic as the number of months will change day by day as we move through the calendar so I'd have to then write a function to figure out the number of months.